cms/envs/common.py

```diff
@@ -208,6 +208,12 @@
 
     # Show Language selector
     'SHOW_LANGUAGE_SELECTOR': False,
+
+    # Temporary feature flag for disabling saving of subsection grades.
+    # There is also an advanced setting in the course module.  This
+    # feature flag is used only to toggle whether the advanced setting
+    # appears in Studio.
+    'ENABLE_SUBSECTION_GRADES_SAVED': False,
```

Comment:
So as I understand it, the settings-level feature flag is set here for CMS, and that allows courses to optionally opt in to the new system.

Should we also add/respect this setting in the LMS, in case we want to disable the new system due to buggy behavior?